code-challenges.test.js

```diff
@@ -21,14 +27,31 @@ const fibonacciLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+const fibonacciFunc = (number) => {
+  // first two numbers of the array that will begin the fibonacci sequence
+  const fibonacciSeq = [1, 1]
+  // Starting a for loop at index 2 iterating up to the parameter "number" that will be the length of the array
+  for (let i = 2; i < number; i++) {
+    // start the fibonacci sequence by adding the last two numbers in the array
+    const nextValue = fibonacciSeq[i - 1] + fibonacciSeq[i - 2]
+    // push the calculated number to the end of the array
+    fibonacciSeq.push(nextValue)
+  }
 
+  return fibonacciSeq;
+}
```

Comment:
I saw your comment about the Fib problem, which you wrote very well.  I'm happy to meet with you one on one to go over it. We will definitely look at it more closely on Friday in the Assessment Review. Just let me know!